.github/workflows/node.yaml

```diff
@@ -0,0 +1,38 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04, macos-latest, windows-latest]
```

Comment:
@manekenpix what is our OS for staging and deploy? If we can use that here, it would be great.

Reply:
Fedora 30

Reply:
Matching to utilize Fedora 30, 32 (which was just released and many would upgrade to that)